node-repository/src/test/java/com/yahoo/vespa/hosted/provision/restapi/v2/responses/node4-after-changes.json

```diff
@@ -39,6 +39,7 @@
   "failCount": 0,
   "hardwareFailure": true,
   "hardwareFailureType": "memory_mcelog",
+  "hardwareFailureDescription": "memory_mcelog",
```

Comment:
Could you add a test that verifies a node with only hardwareFailureType is output with JSON hardwareFailureDescription/-Type?